### ISSUE TITLE
part 17: My debug test window won't hide

### ISSUE THREAD

**Issue by Chriswilldo** [thedigicraft/Atom.CMS#221]
Even though all the coding is presumptively right, i can't hide that damn window or get my footer to color gray. Please reply for more details

**Comment by creptor** [comment]
That would be based on the identifiers given to those elements in the html, check that they're consistent with what you have in the CSS sheet.

To check that your JavaScript is working properly please place your code here.

If you're not sure about something please provide your code, I'll do my best to fix them. (please use a [gist](https://gist.github.com/) for this, just make it public and paste the link here)

**Comment by Chriswilldo** [comment]
css.php
@@ -0,0 +1,30 @@
+		
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
+
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+
+<style>
+	/* Sticky footer styles
+-------------------------------------------------- */
+html, 
+body {
+  height: 100%;
+}
+
+#wrap {
+  min-height: 100%;
+  height: auto;
+  margin 0 auto -60px; padding: 0 0 60px;}
+
+}
+
+#footer { height: 60px; background-color: #f5f5f5; }
+
+#btn-debug { position: absolute; };
+
+</style> 
View
25  data.php
@@ -0,0 +1,25 @@
+<?php 
+
+function data_page($dbc, $id) {
+	
+	$q = "SELECT * FROM pages WHERE id = $id";
+$r = mysqli_query($dbc, $q);
+$data = mysqli_fetch_assoc($r);
+
+$data['body_nohtml']= strip_tags($data['body']);
+
+if($data['body'] == $data['body_nohtml']) {
+
+$data['body_formatted'] = '<p>'.$data['body'].'</p>';	
+	
+} else {
+	
+	$data['body_formatted'] = $data['body'];
+		
+}
+
+return $data;
+
+}
+
+?> 
View
48  index.php
@@ -0,0 +1,48 @@
+<?php 
+include('config/setup.php'); 
+
+
+
+?>
+
+<!DOCTYPE html>
+<html>
+	
+	
+<head>	
+	<title><?php echo $page['title'].' | '.$site_title; ?></title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<?php include('config/css.php'); ?>
+<?php include('config/js.php'); ?>
+
+</head>
+<body>
+<div id="wrap">	
+	
+<?php include(D_TEMPLATE.'/navigation.php'); ?>
+
+<div class="container">
+	<h1><?php echo $page['header']; ?></h1>
+	
+	<?php echo $page['body_formatted']; ?>
+	
+	<?php if(isset($_GET['debug']) == 1) { ?> 
+<pre>
+<?php print_r($page); ?>
+</pre>
+<?php } ?>
+
+</div>
+
+</div>
+
+<?php include(D_TEMPLATE.'/footer.php'); ?>
+
+<div id="console-debug">
+	Test debug window
+</div>
+
+</body>
+	
+</html> 
View
20  js.php
@@ -0,0 +1,20 @@
+<?php //javascript ?>
+
+
+<script src="//code.jquery.com.com/jquery-3.1.0.min.js"</script>
+
+<script src="//code.jquery.com.com/ui/1.12.1/jquery-ui.js"</script>
+
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+
+<script>
+	
+	$(document).ready(function() { 
+		
+		$("#console-debug").hide();
+		
+		});
+	
+</script 
View
13  navigation.php
@@ -0,0 +1,13 @@
+<nav class="navbar navbar-default" role="navigation">	
+	
+	<button id="btn-debug" class="btn btn-success">D-BUD</button>
+	
+<div class="container">	
+<ul class="nav navbar-nav">
+	
+	<?php nav_main($dbc, $pageid)?> 
+
+	<li><a href="#">faq</a></li>
+	<li><a href="#">contact</a></li>
+</ul></div>
+</nav>

**Comment by Chriswilldo** [comment]
https://gist.github.com/Chriswilldo/3d889adb4be726bc54184f8a1c86b6f1

**Comment by Chriswilldo** [comment]
I don't know why this worked but all I did was re copy and paste the sticky footer css and voila it stuck. Now for the debug button..

**Comment by creptor** [comment]
well I have found it :D was on phone later so didn't realized until now....
On line 20 of the "js.php" you should change the following:
```
</script
```
with:
```
</script>
```

**Comment by creptor** [comment]
ALSO IMPORTANT...
You shouldn't mix PHP with JavaScript, changing that file to "js.js" makes the code less server intensive and faster. Also it keeps the file consistent, because if you use PHP there should be mostly server code (or code based logic) and no other (not essential) type of code in the file.

**Comment by Chriswilldo** [comment]
That didn't work. For line 20 that end script connects to the javascript cdn script . Still can't get it to work

**Comment by Chriswilldo** [comment]
I got it! So here's what I did / what happened. So with my footer, ultimately, I got it to properly color and stick by re copying the css from bootstrap, afterwards I looked up how to make it stick to the bottom by adding the codes bottom: 0; after position:absolute; and margin: 0 auto; after height. Then I went to w3schools and compared how they got their button to hide text. That helped me figure out to put my text in a h4 heading enclosed within a div tag. I'm not 100% sure why that works, but it does. Without the div tag, for some reason the text start to sink under the footer and without the header tag it hides the footer and the debug window. I went ahead and put the test debug on another page to test it better. I'm pretty you do that soon anyways though in your series. Hopefully the toggle part will work just as successfully.
